packages/relay/src/lib/constants.ts

```diff
@@ -70,5 +70,92 @@ export default {
+    ETH_GET_TRANSACTION_BY_HASH: 'eth_getTransactionByHash',
+    ETH_FEE_HISTORY: 'eth_feeHistory',
+    ETH_GET_STORAGE_AT: 'eth_getStorageAt',
+    ETH_GET_CODE: 'eth_getCode',
+    ETH_SIGN_TRANSACTION: 'eth_signTransaction',
+    ETH_SIGN: 'eth_sign',
+    ETH_PROTOCOL_VERSION: 'eth_protocolVersion',
+    ETH_SEND_TRANSACTION: 'eth_sendTransaction',
+    ETH_COINBASE: 'eth_coinbase',
+    ETH_GET_WORK: 'eth_getWork',
+    ETH_SUBMIT_HASH_RATE: 'eth_submitHashrate',
+    ETH_MAX_PRIORITY_FEE_PER_GAS: 'eth_maxPriorityFeePerGas',
+    ETH_SYNCING: 'eth_syncing',
+    ETH_SUBMIT_WORK: 'eth_submitWork',
+    ETH_MINING: 'eth_mining', 
+    ETH_HASH_RATE: 'eth_hashrate',
+    ETH_ACCOUNTS: 'eth_accounts',
+    ETH_GET_UNCLE_COUNT_BY_BLOCK_NUMBER: 'eth_getUncleCountByBlockNumber',
+    ETH_GET_UNCLE_COUNT_BY_BLOCK_HASH:'eth_getUncleCountByBlockHash',
+    ETH_GET_UNCLE_BY_BLOCK_NUMBER_AND_INDEX: 'eth_getUncleByBlockNumberAndIndex',
+    ETH_GET_UNCLE_BY_BLOCK_HASH_AND_INDEX: 'eth_getUncleByBlockHashAndIndex',
+    NET_VERSION: 'net_version',
+    NET_LISTENING: 'net_listening',
+    ETH_CHAIN_ID: 'eth_chainId',
+    ETH_GET_BALANCE: 'eth_getBalance',
+    ETH_BLOCK_NUMBER: 'eth_blockNumber',
+    ETH_GET_LOGS: 'eth_getLogs',
+    ETH_GET_BLOCK_BY_HASH: 'eth_getBlockByHash',
+    ETH_GET_BLOCK_BY_NUMBER: 'eth_getBlockByNumber',
+    ETH_GET_BLOCK_TRANSACTION_COUNT_BY_NUMBER: 'eth_getBlockTransactionCountByNumber',
+    ETH_GET_BLOCK_TRANSACTION_COUNT_BY_HASH: 'eth_getBlockTransactionCountByHash',
+    ETH_GET_TRANSACTION_BY_BLOCK_HASH_AND_INDEX: 'eth_getTransactionByBlockHashAndIndex',
+    ETH_GET_TRANSACTION_BY_BLOCK_NUMBER_AND_INDEX: 'eth_getTransactionByBlockNumberAndIndex',
+    ETH_GET_TRANSACTION_COUNT: 'eth_getTransactionCount',
+    ETH_ESTIMATE_GAS: 'eth_estimateGas',
+    ETH_GAS_PRICE: 'eth_gasPrice',
+    WEB3_SHA: 'web3_sha',
+    NET_PEER_COUNT: 'net_peerCount',
+    ETH_SIGN_TYPED_DATA: 'eth_signTypedData',
+    ETH_GET_PROOF: 'eth_getProof',
+
+    //Events
+    ResponseCode: "ResponseCode",
+    AllowanceValue: "AllowanceValue",
+    ApprovedAddress: "ApprovedAddress",
+    Approved: "Approved",
+    Frozen: "Frozen",
+    KycGranted: "KycGranted",
+    TokenCustomFees: "TokenCustomFees",
+    TokenDefaultFreezeStatus: "TokenDefaultFreezeStatus",
+    TokenDefaultKycStatus: "TokenDefaultKycStatus",
+    TokenExpiryInfo: "TokenExpiryInfo",
+    FungibleTokenInfo: "FungibleTokenInfo",
+    TokenInfo: "TokenInfo",
+    TokenKey: "TokenKey",
+    NonFungibleTokenInfo: "NonFungibleTokenInfo",
+    IsToken: "IsToken",
+    TokenType: "TokenType",
+    Approval: "Approval",
+    PausedToken: "PausedToken",
+    UnpausedToken: "UnpausedToken",
+    CreatedToken: "CreatedToken",
+    MintedToken: "MintedToken",
+    DefaultFreezeStatusChanged: "DefaultFreezeStatusChanged",
+    Transfer: "Transfer",
+
+    CALL_EXCEPTION: "CALL_EXCEPTION",
+    GAS_LIMIT_50_000: { gasLimit: 50_000 },
```

Comment:
Maybe pull this out into a static object `GAS_LIMITS`.
Also this and any constants that are only used in tests should be in a constants file under the `./tests` dir